rosys/vision/camera_provider.py

```diff
@@ -66,6 +67,7 @@ def remove_all_cameras(self) -> None:
             self.remove_camera(camera_id)
 
     def prune_images(self, max_age_seconds: float | None = None):
+        warnings.warn('Pruning images is not required anymore because images is now a deque', stacklevel=2)
```

Comment:
@denniswittich  the standard for deprecation warnings seems to be:
```python
import warnings

warnings.warn('Pruning images is not required anymore because images is now a deque', DeprecationWarning, stacklevel=2)
```
which prints this output:
`sys:1: DeprecationWarning: Pruning images is not required anymore because images is now a deque`
instead of this:
`sys:1: UserWarning: Pruning images is not required anymore because images is now a deque`
However, python3 normally ignores deprecation warnings.
You either have to start it via `python3 -Wd your_program.py` or add this to your file: `warnings.simplefilter('always', DeprecationWarning)`

Reply:
Thanks!